scaladoc/src/dotty/tools/scaladoc/DocContext.scala

```diff
@@ -17,6 +17,8 @@ import java.io.PrintStream
 import scala.io.Codec
 import java.net.URL
 import scala.util.Try
+import scala.collection.mutable
+import dotty.tools.scaladoc.util.Check._
```

Comment:
Where is this import used?

Reply:
Now I've made the import more explicit since it was only one method, not sure why I made it like this before

Reply:
seems like nothing changed here.
There were problems with GitHub last week. Perhaps something didn't get pushed

Reply:
That's true, thanks for noticing, I've force pushed it just now.